content-src/asrouter/schemas/provider-response.schema.json

```diff
@@ -21,6 +21,14 @@
             "description": "An id matching an existing Activity Stream Router template",
             "enum": ["simple_snippet"]
           },
+          "bundled": {
+            "type": "integer",
+            "description": "The number of messages of the same template this one should be shown with (optional)"
+          },
+          "order": {
+            "type": "integer",
```

Comment:
Should this maybe have `"minimum": 0` as well or are we allowing negative values?